README.md

```diff
@@ -51,6 +51,13 @@ Status](https://readthedocs.org/projects/glow/badge/?version=latest)](https://gl
 # Building and Testing
 This project is built using [sbt](https://www.scala-sbt.org/1.0/docs/Setup.html) and Java 8.
 
+To build and run Glow, you must [install conda](https://docs.conda.io/en/latest/miniconda.html) and
+activate the environmet in `python/environment.yml`. 
```

Comment:
Typo: environment

Reply:
Let's also make a note that when `environment.yml` is updated, the user should update their environment as well.